scripts/local_env_vm_prepare

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+## Shell Opts ----------------------------------------------------------------
+
+set -o pipefail
+set -xeuo
+
+CIFMW_REPO_PATH=/home/zuul/src/ci-framework
+
+if [[ ! -d "$CIFMW_REPO_PATH" ]]; then
+    git clone https://github.com/openstack-k8s-operators/ci-framework "$CIFMW_REPO_PATH"
+fi
+
+git clone https://github.com/openstack-k8s-operators/install_yamls /home/zuul/src/install_yamls
+
+pushd "$CIFMW_REPO_PATH"
+pip install --upgrade pip setuptools bindep
```

Comment:
iirc this should be launched with `sudo` in order to expose all the things in the system. "bad", but needed. At least, that's what I did in my demo VM.